packages/zoe/src/contractSupport/bondingCurves.js

```diff
@@ -105,23 +93,19 @@ export const calcLiqValueToMint = ({
  * adding liquidity. We require that the deposited ratio of central to secondary
  * match the current ratio of holdings in the pool.
  *
- * @param {Object} params
- * @param params.centralIn - The value of central assets being deposited
- * @param params.centralPool - The value of central assets in the pool
- * @param params.secondaryPool - The value of secondary assets in the pool
- * @param params.secondaryIn - The value of secondary assets provided. If
+ * @param centralIn - The value of central assets being deposited
+ * @param centralPool - The value of central assets in the pool
+ * @param secondaryPool - The value of secondary assets in the pool
+ * @param secondaryIn - The value of secondary assets provided. If
  * the pool is empty, the entire amount will be accepted
  * @returns {number} - the amount of secondary required
  */
-export const calcSecondaryRequired = ({
+export const calcSecondaryRequired = (
   centralIn,
   centralPool,
   secondaryPool,
   secondaryIn,
-}) => {
-  assertDefined('centralIn', centralIn);
```

Comment:
I think the `assertDefined` are still helpful regardless of whether it is an object parameter or individual parameters. Could we keep them?